arkouda/dataframe.py

```diff
@@ -730,23 +730,23 @@ def __init__(self, initialdata=None, index=None, columns=None):
             self.update_nrows()
 
     def __getattr__(self, key):
-        if key not in self.columns:
+        if key not in self.column_names:
```

Comment:
should this be `self._column_names`

Reply:
We could do it that way, and I think that's how I had it before.  If we do it that way, then every time you loop over the column names you would have to either call df._columns (hidden attribute) or df.columns.values, which creates an index, sends the data to arkouda server, only immediately to pull all the values back, which seemed kind of inefficient.  The column names are small now, but keep in mind, could be more voluminous once we start implementing features like on-hot-encoding.  But I'm open to whatever everyone can agree with.

Reply:
Actually, you would have to call df.columns.values.tolist() to loop over the columns b/c in arkouda indexes store the value as pdarrays rather than lists.

Reply:
I made the changes we discussed.  Let me know what you think.